WordPress/WordPressTest/Blog Dashboard/Cards/DashboardJetpackSocialCardCellTests.swift

```diff
@@ -115,6 +115,27 @@ class DashboardJetpackSocialCardCellTests: CoreDataTestCase {
         // Then
         XCTAssertEqual(subject.displayState, .none)
     }
+
+    // MARK: Atomic Site Tests
+
+    // In some cases, atomic sites could sometimes get limited sharing from the API.
+    // We'll need to ignore any sharing limit information if it's a Simple or Atomic site.
+    // Refs: p9F6qB-dLk-p2#comment-56603
+    func testCardOutOfSharesDoesNotDisplayForAtomicSites() throws {
```

Comment:
❤️  Thanks for adding this test.

💡 Would it make sense to also add a test for the case where the card is shown because the blog is out of shares and is not WPCom nor Atomic to assert that case still works as expected? I noticed that no other tests check for the out of shares card being shown.

Reply:
Good suggestion 👍🏼. I've added a test case for this in https://github.com/wordpress-mobile/WordPress-iOS/pull/22303/commits/f2d49c0e29c6786bcff7990614fc4e31493c18e7 🙂